src/coreclr/CMakePresets.json

```diff
@@ -45,6 +45,21 @@
         }
       }
     },
+    {
+      "name": "linux-base",
+      "hidden": true,
+      "inherits": "base",
+      "cacheVariables": {
+        "CLR_CMAKE_TARGET_OS": "linux"
+      },
```

Comment:
```
                "CMAKE_C_COMPILER": "clang",
                "CMAKE_CXX_COMPILER": "clang++"
```
Is it using gcc for default? If so, let's default to clang since it's used for most cases.

Reply:
It is unspecified by default, so it would use whatever is discovered by CMake via environment variables, kit etc.. I initially set it that way because it is possible to build with both `gcc` and `clang` but if we want to be explicit to use `clang` then that's fine too

Reply:
Hmm, for future usage in the repo, the choice of compiler may be passed with cmake command line. A user can always create a user preset with additional specification. I'm then OK to leave it unspecified.